library/std/src/keyword_docs.rs

```diff
@@ -166,6 +166,12 @@ mod break_keyword {}
 ///
 /// For more detail on `const`, see the [Rust Book] or the [Reference].
 ///
+/// ## Compile-time blocks
+///
+/// The `const` keyword can also be used to define a block of code that is evaluated at compile time.
+/// This is useful for defining constants that require more complex logic to compute.
```

Comment:
Maybe something like

```rust
/// This is useful for ensuring certain computations are completed before optimizations happen, as well as
/// before runtime. For more details, see the [Reference][const-blocks].
```

Reply:
Done.

Reply:
Sorry, i only meant to change the second sentence. The "The `const` keyword can also be used..." part was good to have, mind adding it back?

Reply:
Done.